expect.go

```diff
@@ -88,6 +88,7 @@ import (
 // Expect is a toplevel object that contains user Config and allows
 // to construct Request objects.
 type Expect struct {
+	noCopy   *noCopy
```

Comment:
It should be a pointer.

Reply:
here there is somewhere the value of the Expect is copied go in expect.go line 435
What can we do ?

Reply:
Oh, good catch.
This is a legit use though, because it does deep copy there. So you can just add `// no lint` to that line.

Reply:
done